doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -0,0 +1,1304 @@
+
+* :ref:`fast_pair_input_device` sample:
+
+  * Renamed the sample from Bluetooth: Fast Pair to :ref:`fast_pair_input_device` and moved it to the :file:`samples/bluetooth/fast_pair` folder.
+  * Added automatic switching to the Fast Pair not discoverable advertising mode with the hide UI indication instead of removing the Fast Pair advertising payload when all bond slots are taken.
+  * Increased the system workqueue stack size (:kconfig:option:`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE`) to ``2048`` to prevent stack overflows right after booting the nRF5340 DK.
+  * Fixed an issue where the sample was unable to advertise in Fast Pair not discoverable advertising mode when it had five Account Keys written.
+
+Bluetooth mesh samples
+----------------------
+
+* :ref:`bluetooth_mesh_sensor_client` sample:
+
+  * Fixed an issue with the sample not fitting into RAM size on the ``nrf52dk_nrf52832`` board.
+
+* :ref:`bluetooth_mesh_light` sample:
+
+  * Removed support for the configuration with :ref:`CMSE enabled <app_boards_spe_nspe_cpuapp_ns>` for :ref:`zephyr:thingy53_nrf5340`.
+
+* :ref:`bluetooth_mesh_light_lc` sample:
+
+  * Added support for Composition Data Pages 1 and 2.
+    Support for Composition Data Pages 1 and 2 has a dependency on Bluetooth mesh 1.1 support.
+  * Fixed an issue where the sample could return an invalid Light Lightness Status message if the transition time was evaluated to zero.
+  * Removed support for the configuration with :ref:`CMSE enabled <app_boards_spe_nspe_cpuapp_ns>` for :ref:`zephyr:thingy53_nrf5340`.
+
+* :ref:`bluetooth_mesh_light_dim` sample:
+
+  * Added support for Composition Data Pages 1 and 2.
+    Support for Composition Data Pages 1 and 2 has a dependency on Bluetooth mesh 1.1 support.
+  * Removed support for the configuration with :ref:`CMSE enabled <app_boards_spe_nspe_cpuapp_ns>` for :ref:`zephyr:thingy53_nrf5340`.
+
+* :ref:`bluetooth_mesh_light_switch` sample:
+
+  * Removed support for the configuration with :ref:`CMSE enabled <app_boards_spe_nspe_cpuapp_ns>` for :ref:`zephyr:thingy53_nrf5340`.
+
+* :ref:`bluetooth_mesh_sensor_server` sample:
+
+  * Added support for Composition Data Pages 1 and 2.
+    Support for Composition Data Pages 1 and 2 has a dependency on Bluetooth mesh 1.1 support.
+  * Added a getter for the :c:var:`bt_mesh_sensor_rel_runtime_in_a_dev_op_temp_range` sensor.
+  * Removed support for the configuration with :ref:`CMSE enabled <app_boards_spe_nspe_cpuapp_ns>` for :ref:`zephyr:thingy53_nrf5340`.
+  * Fixed an issue where the :c:var:`bt_mesh_sensor_time_since_presence_detected` sensor could report an invalid value when the time delta would exceed the range of the characteristic.
+
+* Fixed an issue where some samples copied using the `nRF Connect for Visual Studio Code`_ extension would not compile due to relative paths in :file:`CMakeLists.txt`, which were referencing files outside of the applications folder.
```

Comment:
This bullet is a general comment for mesh samples and should be moved up to right under the section, as the first entry.